src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier(random);
+        FigureSupplier figureSupplier = new FigureSupplier(random, colorSupplier);
+
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < 6; i++) {
+            if (i > 2) {
+                figures[i] = figureSupplier.getDefaultFigure();
+                figures[i].draw();
+                continue;
+            }
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
```

Comment:
If you draw the shape at the same time you create it, do you really need an array?

Reply:
The array was required in the task